File: minimum-bribes.js
function minimumBribes(q) {
    // Write your code here
    let queue = q
    
    let bribes = 0
    
    for (let i = 0; i < queue.length; i++) {
        // console.log(`Q[i]: ${q[i]} and i: ${i}`)
        if (q[i] - (i + 1) > 2) {
            console.log("Too chaotic")
            return
        } 
        
        for (let j = Math.max(0, q[i] - 2); j < i; j++) {
            if (q[j] > q[i]) {
                bribes++
            }
        }
    }
    
    console.log(bribes)
}

File: minimum-bribes.py
def minimum_bribes (q):
    queue = q
    bribes = 0
    
    for index, person in enumerate(queue):
        # print(person)
        if person - (index + 1) > 2:
            print("Too chaotic")
            return
            
        for j in range(max(0, person - 2, index)):
            if queue[j] > person:
                bribes += 1
                
    print(bribes)
